### DISCUSSION TITLE
SSO like old RPort-Plus

### DISCUSSION THREAD

**Discussion by 3gmgithub** [openrport/openrport#26] [Roadmap Feature Wishes Idea]
Really would like an SSO feature, just like the old RPort-Plus...
That would make this a killer project.

**Comment by alhdo** [comment]
We will continue the development. To make this project really opensource we are working on hour own open source version of the ui. Once we reach this milestone we will consider getting back the plugin system as opensource also.

**Comment by 3gmgithub** [comment]
Excellent!  Really thanks again!

**Comment by mdimenco** [comment]
I contributed #71 witch implements SSO as local plugin.
